### DISCUSSION TITLE
Stream PDF file with Livewire

### DISCUSSION THREAD

**Discussion by AggelosGalatis** [livewire/livewire#5422] [1. Help] [answered]
Hello. I would like to ask if is there any way that i can display a PDF file from method in Livewire.

**Comment by joshhanley** [comment, marked as answer]
@AggelosGalatis so because a Livewire request typically is an AJAX request, you can't return a streamed response. The best way it to create a new endpoint with a standard Laravel controller for your streamed PDF, that just returns the pdf. Then in your Livewire component you can just add an `<a>` tag and link to the pdf endpoint.

This has the added bonus that you can make it open the streamed pdf in a new window/tab.

Hope this helps!
